.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
There's a slight irony that you updated the tab spaces to be 4, but this new file is still at tabspace 2 :P

Reply:
In fact (possibly as a separate PR) it seems like a whole-project prettier run might be in order.

Reply:
tabspace 2 forever! 😈